cli/src/semgrep/formatter/text.py

```diff
@@ -573,7 +574,10 @@ def _build_text_output(
                 color_output,
                 per_finding_max_lines_limit,
                 per_line_max_chars_limit,
-                is_same_file,
+                # if we have dataflow traces on, then we should print the separator,
```

Comment:
Unclear from just this comment what "the separator" is; if it's clear from the rest of the code just ignore this

Reply:
removed it